SPECS/libsolv/libsolv.spec

```diff
@@ -77,6 +78,9 @@ find %{buildroot} -type f -name "*.la" -delete -print
 %{_mandir}/man1/*
 
 %changelog
+* Fri Jan 02 2024 Alberto David Perez Guevara <aperezguevar@microsoft.com> - 0.7.28-1
```

Comment:
Fri `Feb` 02?

Reply:
that is when I started working on these changes.

Reply:
In that case the date may need an adjustment - 2nd of January 2024 was a Tuesday. This sounds nit-picky but an incorrect date generates a warning message during builds and we just want to avoid them.

Reply:
Yes, an incorrect day/date match will generate a warning. If this was in Jan, it should be `Tue Jan 02`

Reply:
I see, thanks for verifying and the explanation, I'll make the changes according to the feedback.